src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebReportRouteConfig.kt

```diff
@@ -4,17 +4,25 @@ import org.vaccineimpact.orderlyweb.WebEndpoint
 import org.vaccineimpact.orderlyweb.app_start.RouteConfig
 import org.vaccineimpact.orderlyweb.controllers.api.ArtefactController
 import org.vaccineimpact.orderlyweb.controllers.web.ReportController
+import org.vaccineimpact.orderlyweb.json
 import org.vaccineimpact.orderlyweb.secure
+import spark.route.HttpMethod
 
 object WebReportRouteConfig : RouteConfig
 {
     private val readReports = setOf("report:<name>/reports.read")
+    private val reviewReports = setOf("*/reports.review")
     override val endpoints = listOf(
             WebEndpoint("/reports/:name/:version/",
                     ReportController::class, "getByNameAndVersion")
                     .secure(readReports),
             WebEndpoint("/reports/:name/versions/:version/artefacts/:artefact/",
                     ArtefactController::class, "getFile")
-                    .secure(readReports)
+                    .secure(readReports),
+            WebEndpoint("/reports/:name/versions/:version/publish/",
+                    org.vaccineimpact.orderlyweb.controllers.api.ReportController::class, "publish",
+                    method = HttpMethod.post)
```

Comment:
Need a test for this being locked down to reviewers.

Reply:
https://github.com/vimc/orderly-web/pull/45/files#diff-0a3fddc91043e8c897a7e5dc942093b8